test/FunctionsV2/CorrelationEndToEndTests.cs

```diff
@@ -405,14 +405,6 @@ private static List<OperationTelemetry> GetCorrelationSortedList(OperationTeleme
             var result = new List<OperationTelemetry>();
             if (current.Count != 0)
             {
-                foreach (var some in current)
-                {
-                    if (parent.Id == some.Context.Operation.ParentId)
-                    {
-                        Console.WriteLine("match");
```

Comment:
writing to console in GitHub actions is flakey and can cause runtime errors. So I removed this, which was only there for debugging purposes